packages/ui/client/composables/error.ts

```diff
@@ -1,4 +1,6 @@
 import Filter from 'ansi-to-html'
+import type { ErrorWithDiff } from 'vitest'
+import { parseStacktrace } from '../../../utils/src/source-map'
```

Comment:
Should this be imported from `@vitest/utils/source-map`? Currently this is directly referencing typescript file in `packages/utils/src/source-map.ts`.

Probably client js will be bundled in the same way, but it might be odd that since `packages/ui` has `"@vitest/utils": "workspace:*"` dependency explicitly.

Reply:
Whole UI is bundled so it doesn't really matter

Reply:
Interestingly enough, changing to `@vitest/utils` makes bundler a bit smaller